# This message describes a joint movement

# The angle the joint is currently at
float64 joint_angle

# The current velocity of the joint
float64 joint_velocity

# The acceleration the joint currently experiences
float64 joint_acceleration

# The torque the joint is delivering
float64 joint_torque

# The current temperature of the joint
float64 joint_temperature